oak/server/oak_manager.cc

```diff
@@ -55,6 +62,12 @@ void OakManager::CreateEnclave(const std::string& node_id, const std::string& mo
   ::oak::InitializeInput* initialize_input = config.MutableExtension(::oak::initialize_input);
   initialize_input->set_node_id(node_id);
   initialize_input->set_module(module);
+
+  // Uses a random port number in range of [node_port_min, node_port_max].
+  // TODO: keep track of "free" ports.
+  initialize_input->set_grpc_port(FLAGS_node_port_min +
+      std::rand() % (FLAGS_node_port_max - FLAGS_node_port_min + 1));
```

Comment:
The use of std::rand is just a hack to test any port in the published range should work; to be replaced with something more robust (maybe a bitset to track used/unused state for each port number, also testing if a port is actually open before sending it to enclave manager)